### ISSUE TITLE
My PS4 controller is detected as an Xbox 360 controller

### ISSUE THREAD

**Issue by ItZRakoon** [Jays2Kings/DS4Windows#481]
After installing the DS4Windows for windows 7 and the Xbox 360 Drivers I can´t edit the PS4 controller, this screen shows up instead.

https://gyazo.com/71984d1c4df89f5589f94f870ec34568

**Comment by Yohoki** [comment]
This is correct. Ds4Windows makes the computer think your controller is an Xbox controller. Any programs with Xbox controller support will work fine with the "Default" options like this. If you select to create or edit a profile, you can then click a button on your controller and the screen pops up that you have shown. In this screen, you can assign a keyboard button or mouse input to that button.

**Comment by ItZRakoon** [comment]
So if I bind a keyboard key to "A" from the X-box controller, will the "X" from the PS4 controller bind that key to it?

**Comment by Yohoki** [comment]
Yes. Bring up the editing window and press "X" on your PS4 controller. It will pop up that screen with the Xbox controller, a keyboard and a mouse. You can either click one of the icons or hit a key on your keyboard and it will bind that to your "X" button on the ps4 controller. Save the settings and open Notepad to test out if it's working. The reason the Xbox controller is there is so that you can switch a ps4 button to an xbox button. The mouse is also there, so you could do things like "Left stick" as mouse movement and "X" and "left click". I do not currently have a mouse, so that works great for me. :D
